subgraphs/uniswap-v3-forks/src/mappings/position-manager.ts

```diff
@@ -157,9 +166,22 @@ export function handleDecreaseLiquidity(event: DecreaseLiquidity): void {
 }
 
 export function handleTransfer(event: Transfer): void {
-  const account = getOrCreateAccount(event.params.to);
   const position = getOrCreatePosition(event, event.params.tokenId);
 
+  const account = getOrCreateAccount(event.params.to);
+  const oldAccount = getOrCreateAccount(position!.account);
+
+  account.positionCount += INT_ONE;
```

Comment:
Is it possible for a transfer to not create a new position, but to just add to an existing one?

Reply:
Yeah, this operation should not create any new positions. I decided to handle this as just transferring the position from one account to the other and incrementing this value. Along with the open and closed position count depending on if it is opened or closed.

Reply:
I see, I missed the part that it was in an if block that only executed if the account was created